examples/hyperbee.js

```diff
@@ -33,10 +29,13 @@ rawCandleData = rawCandleData.sort((a, b) => {
   return a[0] - b[0]
 })
 
-const kp = HFS.candleMarketDataKey({
+const market = {
   symbol: SYMBOLS.BTC_USD,
   tf: TIME_FRAMES.ONE_HOUR
-})
+}
+
+// simulate dazaar market data 5min candles
+const kp = 'c!5m!'
```

Comment:
`HFS.candleMarketDataKey(market)` instead of `kp = 'c!5m!'` matbe?

Reply:
yeah makes sense, will update dazaar-market-data to those keys, so we have uniformity across repos